homeworks/hw_3.py

```diff
@@ -0,0 +1,26 @@
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from sqlalchemy import Column, Integer, String, Numeric, Boolean, ForeignKey
+from sqlalchemy.ext.declarative import declarative_base
```

Comment:
Это устаревший вариант импорта. Теперь нужно так 
```suggestion
from sqlalchemy.orm import declarative_base
```